UnitTests/Utilities/GeometryModelRenderer/src/PointCloudGeometryModelRenderer.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+	Copyright (c) 2021 Technical University of Munich
+	Chair of Computational Modeling and Simulation.
+
+	TUM Open Infra Platform is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License Version 3
+	as published by the Free Software Foundation.
+
+	TUM Open Infra Platform is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "PointCloudGeometryModelRenderer.h"
+
+PointCloudGeometryModelRenderer::PointCloudGeometryModelRenderer(const buw::ReferenceCounted<buw::IRenderSystem>& renderSystem)
+	: GeometryModelRenderer::GeometryModelRenderer(renderSystem)
+{
+	offGeometryEffect_ = buw::makeReferenceCounted<oip::OffGeometryEffect>(
```

Comment:
PointCloudGeometryEffect in the whole file

Reply:
Yes, here it is corrected: [95909e5](https://github.com/tumcms/Open-Infra-Platform/pull/525/commits/95909e54553273d5aa08fde92d8876854ad61821)